org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/client/AbstractXMLNotification.java

```diff
@@ -0,0 +1,34 @@
+package org.eclipse.lemminx.client;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import org.eclipse.lemminx.customservice.ActionableNotification;
+import org.eclipse.lemminx.customservice.XMLLanguageClientAPI;
+import org.eclipse.lemminx.settings.SharedSettings;
+import org.eclipse.lsp4j.Command;
+import org.eclipse.lsp4j.MessageParams;
+import org.eclipse.lsp4j.MessageType;
+
+public abstract class AbstractXMLNotification {
```

Comment:
I think it would make more sense to call this class `AbstractXMLNotifier`, since this generates a notification, but doesn't represent one itself.